langgraph/src/graph/state.ts

```diff
@@ -53,19 +94,34 @@ export interface StateGraphArgs<Channels extends object | unknown> {
 }
 
 export class StateGraph<
-  State extends object | unknown,
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  Update extends object | unknown = Partial<Record<keyof State, any>>,
+  SD extends StateDefinition | unknown,
```

Comment:
I generally don't like abbreviations but it's fine

Reply:
I did this to avoid this being same name as the name the user would likely use in their code, which I think could be a little confusing